karman-avionics/src/tasks/Background.h

```diff
@@ -10,23 +10,23 @@
 
 #ifndef BACKGROUND_H_
 #define BACKGROUND_H_
+
 #include <compiler.h>
+
+/** Size of backround function array */
 #define MAX_BACKGROUND_FUNCS (20)
 
+/** Custom return type for backround functions */
 typedef enum {
     BKGND_FUNC_SUCCESS,
     BKGND_FUNC_FAILURE,     
 } background_status_t;
 
+/** Type defintion to keep all backround functions organized */
 typedef void (*background_func_t)(void);
 
 void background_task_func(void);
 
-/* Utility to register a background function */
```

Comment:
Would it be worth adding in some kind of commenting here even if it isn't the previous version?

Reply:
The same comment is now in Background,c. Having functions commented in only one place makes it easier to avoid conflicting information